src/Controls/Internal/TextCopyPasteHelper.ts

```diff
@@ -0,0 +1,87 @@
+﻿module Fayde.Controls.Internal {
+    export class TextCopyPasteHelper {
+        private callback: any = false;
+        private pastedText: string = "";
+
+		GetPastedText(): string {
+            return this.pastedText;
+        }
+
+        private createElement(text: string): HTMLDivElement {
+            (<any>document.body).scroll = "no";
+            document.documentElement.style.overflow = "hidden";
+            var div = document.createElement("div");
+            div.id = "special_copy";
+            div.style.left = "-1000px";
```

Comment:
The previous comment may be because the div is off-screen.  We can leave it on-screen, but just make `opacity: 0.0`.  See https://github.com/wsick/minerva/blob/zoom/src/zoom.ts#L26-L31 for existing usage.